### DISCUSSION TITLE
handbook/test-runners/protractor/

### DISCUSSION THREAD

**Discussion by giscus** [serenity-js/serenity-js#1796] [Comments]
# handbook/test-runners/protractor/

Protractor is an end-to-end test framework for Angular and AngularJS applications, based on Selenium 3.

https://serenity-js.org/handbook/test-runners/protractor/

**Comment by JackCurtin-eaton** [comment]
Yes, a tutorial on migrating from protractor to webdriverio would be great!

**Reply by jan-molak** [reply]
Hey @JackCurtin-eaton! Sure thing, it's now available at https://serenity-js.org/handbook/getting-started/serenity-js-with-protractor/

Let me know what you think
